earth2mip/inference_ensemble.py

```diff
@@ -258,7 +258,19 @@ def perturb(x, rank, batch_id, device):
             )
         if rank == 0 and batch_id == 0:  # first ens-member is deterministic
             noise[0, :, :, :, :] = 0
-        x += noise
+
+        if config.ic_perturbed_channels == "All":
```

Comment:
make this case invariant... so `config.ic_perturbed_channels.lower() == "all"`

Reply:
--> 262 if config.ic_perturbed_channels.lower() == "all":
    263     x += noise
    264 else:

AttributeError: 'list' object has no attribute 'lower'